API/README.md

```diff
@@ -31,9 +31,8 @@ is provided and decide if it can proceed with the input given.
 - **page_size** - integer - Split the results into pages with this number of results each (e.g.:  20)                                                                                       
 - **page_number** - integer - Page number of results when the number of results exceeds the page_size (e.g.:  1)                                                                            
 - **reasoner_ids** - array - List of reasoners to consult for the query (e.g.:  [ "RTX", "Robokop" ])                                                                                       
-- **query_message** - object - Message object that represents the query to be answered (e.g.: )                                                                                             
-- **previous_message_processing_plan** - object - Container for one or more Message objects or identifiers for one or more Messages along with a processing plan for how those messages should be processed and returned (e.g.: )                                                                                                                                                       
-
+- **query_message** - object - Message object that represents the query to be answered (e.g.: ) which contains a [query graph representing the structure of a query](https://github.com/NCATS-Tangerine/NCATS-ReasonerStdAPI/blob/master/API/TranslatorReasonersAPI.yaml#L664).                                                                                           
+- **previous_message_processing_plan** - object - Container for one or more Message objects or identifiers for one or more Messages along with a processing plan for how those messages should be processed and returned (e.g.: ). Contains embedded **Message** objects similar to the ones referenced by **query_message** above), which also contain a [query graph object representing the structure of a query](https://github.com/NCATS-Tangerine/NCATS-ReasonerStdAPI/blob/master/API/TranslatorReasonersAPI.yaml#L664)                                                                                                                                                 
```

Comment:
There may be a missing parenthesis here: `(similar...`

Reply:
Fixed. Took the opportunity to move things around a bit.